compiler+runtime/src/cpp/jank/runtime/obj/persistent_vector_sequence.cpp

```diff
@@ -18,7 +18,11 @@ namespace jank::runtime
   /* behavior::objectable */
   native_bool obj::persistent_vector_sequence::equal(object const &o) const
   {
-    return detail::equal(o, vec->data.begin(), vec->data.end());
+    return detail::equal(
```

Comment:
I spotted these and fixed them on my clang 19 branch, since they were leading to hard crashes. :facepalm: Also led to iterations on your PR. My own bugs. We can keep this and I'll just resolve the conflict when clang 19 merges.